tests/smoke/features/shareFileJourney.feature

```diff
@@ -7,38 +7,50 @@ Background:
-  Then "Alice" creates following folders
-    | folder_to_shared           |
-    | folder_to_share1/subFolder |
+  When "Alice" opens the "files" app
+  And "Alice" navigates to the files page
+  And "Alice" creates following folders
+    | folder_to_shared |
   And "Alice" uploads following resources
     | resource  | to               |
-    | lorem.txt | folder_to_shared |             
-  Then "Alice" moves following resources
-    | resource                             | to                         |
-    | folder_to_shared/lorem.txt           | folder_to_share1/subFolder |
-    | folder_to_share1/subFolder/lorem.txt | All files                  |
-    | lorem.txt                            | folder_to_shared           |
-  Then "Alice" copies following resources
-    | resource                             | to                         |
-    | folder_to_shared/lorem.txt           | folder_to_share1/subFolder |
-    | folder_to_share1/subFolder/lorem.txt | All files                  |
-    | lorem.txt                            | folder_to_share1           |
-  Then "Alice" shares following resources
+    | lorem.txt | folder_to_shared | 
+  Then "Alice" checks whether the following resources exist
+    | folder_to_shared/lorem.txt |
+  And "Alice" shares following resources
     | resource         | user  | role   |
     | folder_to_shared | Brian | editor |
   Given "Brian" has logged in
-  Then "Brian" opens the "files" app
-  Then "Brian" accepts following resources
+  When "Brian" opens the "files" app
+  And "Brian" accepts following resources
     | folder_to_shared |
-  Then "Brian" renames following resource
-    | resource                              | as                   |
-    | Shares/folder_to_shared               | folder_to_shared_new |
-    | Shares/folder_to_shared_new/lorem.txt | loremNew.txt         |
+  And "Brian" renames following resource
+    | resource                          | as            |
+    | Shares/folder_to_shared/lorem.txt | lorem_new.txt |
   And "Brian" uploads following resources
-    | resource   | to                          |
-    | simple.pdf | Shares/folder_to_shared_new | 
+    | resource   | to                      |
+    | simple.pdf | Shares/folder_to_shared | 
+  And "Brian" copies following resources
+    | resource                 | to        |
+    | Shares/folder_to_shared  | All files |
+  And "Brian" has logged out
+  When "Alice" opens the "files" app
+  Then "Alice" checks whether the following resources exist
+    | folder_to_shared/lorem_new.txt |
+    | folder_to_shared/simple.pdf    |
+  When "Alice" creates new versions of the folowing files
+    | resource   | to               |
+    | simple.pdf | folder_to_shared |   
+  Then "Alice" checks that new version exists
+    | folder_to_shared/simple.pdf |
+  When "Alice" removes following resources
+    | folder_to_shared/lorem_new.txt |
+    | folder_to_shared               |
+  Given "Brian" has logged in
```

Comment:
see above, we do not need to log out "brian", then no log in is needed anymore

Reply:
deleted row